LondonBritishLibrary/orient/BLorient11620.xml

```diff
@@ -0,0 +1,168 @@
+                            <title ref="LIT7334Nahabt"/>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="omitted"/> ጸሎት፡ በእንተ፡ ነሀብት፡ አስማተ፡ ሰሎሞን፡ ዘረበቦሙ፡ ለአጋንንት፡ ከመ፡ መርበብ፡ ዘዓሣ፡
+                                ዘባሕር፡ እንዘ፡ ይብል፡ ሰደቃኤል፡ ጸደቃታኤል፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <title ref="LIT6609ProtectPrayer"/>
+                            <note>Prayer against demons and about the healing of a man possessed with a demon, containing verses from 
+                                <ref type="work" corresp="LIT1560Gospel">Mt. 9:32-4</ref>.</note>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <title ref="LIT4620Prayer"/>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="omitted"/> ጸሎት፡ በእንተ፡ አላሁማ፡ ወያኑራሽም፡ ዲጂ፡ <gap reason="omitted"/>
+                                በኃይለ፡ ዝንቱ፡ አስማቲከ፡ ዘረበቦሙ፡ ለአጋንንት፡ ከመ፡ መርበብተ፡ ዓሣ፡ ዘባሕር፡ ወይቤ፡ ሰሎሞን፡ ሎፍሐም፡ ሎፍሐም፡ ሎፍሐም፡ <gap reason="omitted"/>
+                                በኃይለ፡ ዝንቱ፡ አስማቲከ፡ አድኅኖ፡ እምሕማመ፡ ነሀብት፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <title ref="LIT7276MarkOne"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <title ref="LIT000000000000000000">Prayer for undoing charms, maftǝḥe śǝrāy</title>
+                            <incipit xml:lang="gez">አኽያ፡ ሹራህያ፡ አልሻዳይ፡ <gap reason="omitted"/> በኃይለ፡ ዝንቱ፡ አስማቲከ፡ ፍታህ፡ ሥራዮሙ፡ 
+                                ለመሠርያ<supplied reason="undefined" resp="PRS8999Strelcyn">ን</supplied>፡ እምላዕለ፡ ለ<gap reason="illegible"/>
+                                <gap reason="omitted"/> በከመ፡ ፈታህኮሙ፡ ለጴጥሮስ፡ ወለጳውሎስ፡ ወፈታህከ፡ ውስተ፡ ቴርም፡ ሥራየ፡ ቆጵርያኖስ፡ ከማሁ፡ ፍታህ፡ ሥራየ፡ ብእሲት፡
+                                ወብእሲት፡ ሥራየ፡ ባርያ፡ ወአጋንንት፡ አቅጥናኤል፡ ክፍትናኤል፡ <gap reason="omitted"/> በኃይለ፡ ዝንቱ፡ <!-- Strelcyn added: (MS. ''ዝ sic) -->
+                                አስማቲከ፡ ዘርዝር፡ ወርጽሽጅ፡ ዝንቱ፡ <sic resp="PRS8999Strelcyn">ሥራይ፡</sic> በእሲ፡ ወብእሲት፡ ሥራየ፡ ማሪ፡ ወማሪት፡</incipit>
+                            <explicit xml:lang="gez"></explicit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Scroll">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1</measure>
```

Comment:
`  <measure unit="leaf" quantity="1">1</measure>` is better

Reply:
It's not a problem (it's present in most older records), just for the future for better filtrability